src/ComponentInstaller.php

```diff
@@ -543,9 +544,12 @@ private function promptToRememberOption(InjectorInterface $injector, $packageTyp
         $ask = ["\n  <question>Remember this option for other packages of the same type? (Y/n)</question>"];
 
         while (true) {
-            $answer = strtolower($this->io->ask(implode($ask), 'y'));
+            $answer = $this->io->ask(implode($ask), 'y');
+            if (! is_string($answer)) {
```

Comment:
This should be an assertion: it should never be a non-string, so a silent failure is not really OK